__tests__/authors.test.js

```diff
@@ -0,0 +1,38 @@
+const setup = require('../data/setup');
+const app = require('../lib/app');
+const pool = require('../lib/utils/pool');
+const request = require('supertest');
+const Author = require('../lib/models/Author');
+
+describe('authors routes', () => {
+  beforeEach(() => {
+    return setup(pool);
+  });
+
+  it('should return a list of authors with nested books', async () => {
+    const results = await request(app).get('/authors');
+    expect(results.body.length).toEqual(2);
+  });
```

Comment:
Possibly consider revising this test, as if data is added, the test will fail. Consider testing the shape of the data with .toEqual(expect.arrayContaining([{...}])